README.md

```diff
@@ -1,11 +1,23 @@
-<img src="https://media2.giphy.com/media/Nx0rz3jtxtEre/giphy.gif?cid=ecf05e47k1el00v2wbzxdaemqy93cxosmaf6bp2r9trzlc03&rid=giphy.gif&ct=g"/>
+# `POKEDEX PICKER`
 
-If you're looking for a job, head on over to [PROMPT.md](./PROMPT.md).
+## Description
+Allows the user to scroll through Pokemons either one-by-one or selecting from a list in order to gather or remove favorites. The technologies used to achieve this were React for reusable components, Chakra-UI for styles, Node.js and Express.js to implement and use APIs, and postgreSQL to store favorites inside a database.
 
+https://user-images.githubusercontent.com/46462294/154218582-387e9800-6b5c-407e-93ba-0376d54ce027.mp4
 
-</br></br></br></br>
+## How to Install and Run the Project
+1. After cloning the project, delve into both _server_ and _client_ folders in order to `npm i` or `yarn install` to download the libraries.
+2. To run the server, head into the server directory and type `node server.js`.
+3. To run the client, head into the client directory and type `yarn start` or `npm start`.
 
-<!-- Delete this line and everything above it -->
+**Now I know what you are thinking...the cards are blank. That is because I used an env to shield sensitive information regarding the database I have used.**
 
-# `Your App Name Here`
-Add some pretty documentation here that tells us how to start your app so we can see the awesome thing you built!
+4. Make sure to have postgreSQL installed in order to complete the next step. This is optional if you wish to skip storing favorites for Pokemon. In order to skip this step, you must head over to _pokeDexFavorites.js_ under routes folder inside server directory and comment out the routers inside.
+
+5. Create an env file that would fill in the info where you would see `process.env` in _database.js_ located inside server folder. The info needed includes the following:\
```

Comment:
I have used a package called `dotenv` to access the env.

If you'd like, you can find out more about the package here: https://www.npmjs.com/package/dotenv

Reply:
Ah I totally missed that you imported `dotenv` in another part of the project; my recommendation is to do that wherever your app starts, so `server/index.js`, but mostly because that's often the first place devs look when opening up a new project 🙂 

The other recommendation I'd make is to create a `.env.sample` file which is _**not**_ excluded via your `.gitignore`. When you follow that pattern, a dev just has to copy + rename the file and then provide values 🎉

Reply:
Would the .env.sample you mentioned look like this? https://github.com/maherhalabi/pokedex-picker/commit/8b25a105a2d2271f1eb44cc8c6493b269c99458d

Reply:
Perfect!